content/posts/2020-06-01-hello-world/index.md

```diff
@@ -11,7 +11,7 @@ comments: true
 
 In 2014, Loadsmart was born. At that time, we had a lot of ideas around what we wanted to achieve and the products we could build to solve the many challenges facing logistics companies. We knew our impact could disrupt the industry. To this day, many logistics processes are still completed by repetitive tasks, phone calls, and an extensive list of fragmented touch points, but, thanks to the decisions we made in 2014, we are helping the industry embrace technology and understand that it is the only option to scale efficiently in 2020.
 
-We were a small yet excited team sharing a tiny little office in New York City, on one of the busiest avenues in Midtown Manhattan. We would bump each others’ chairs a couple of times a day when coming back to our desks after a meeting or an always pleasant team lunch (we miss [Morocho](https://www.yelp.com/biz/morocho-peruvian-fusion-new-york) so much!).
+We were a small yet excited team sharing a tiny little office in New York City, on one of the busiest avenues in Midtown Manhattan. We would bump each others’ chairs a couple of times a day when coming back to our desks after a meeting or an always pleasant team lunch (we miss [Morocho](https://www.yelp.com/biz/morocho-peruvian-fusion-new-york-4) so much!).
```

Comment:
This was the output from the textlint, the original content moved permanently to this url.